apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,399 @@
+                      ? html`
+                          <button
+                            class="btn btn-xs mr-1 js-sync-popover"
+                            data-toggle="popover"
+                            data-trigger="hover"
+                            data-container="body"
+                            data-html="true"
+                            data-title="Sync Warnings"
+                            data-content='<pre style="background-color: black" class="text-white rounded p-3 mb-0">${question.sync_warnings_ansified}</pre>'
+                          >
+                            <i
+                              class="fa fa-exclamation-triangle text-warning"
+                              aria-hidden="true"
+                            ></i>
+                          </button>
+                        `
+                      : ''}
+                  ${question.display_name}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/topic'); %>", {
+                    topic: question.topic,
+                  })}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/tags'); %>", {
+                    tags: question.tags,
+                  })}
+                </td>
+                <td>
+                  ${maxPoints({
+                    max_auto_points: question.max_auto_points,
+                    max_manual_points: question.max_manual_points,
+                    points_list: question.points_list,
+                    init_points: question.init_points,
+                  })}
+                </td>
+                <td>${question.max_manual_points || '—'}</td>
+                ${showAdvanceScorePercCol
+                  ? html`
+                      <td
+                        class="${question.assessment_question_advance_score_perc === 0
+                          ? 'text-muted'
+                          : ''}"
+                        data-testid="advance-score-perc"
+                      >
+                        ${question.assessment_question_advance_score_perc}%
+                      </td>
+                    `
+                  : ''}
+                <td>
+                  ${question.mean_question_score
+                    ? `${question.mean_question_score.toFixed(3)}`
+                    : ''}
+                </td>
+                <td>
+                  ${question.avg_question_score_perc
+                    ? `${question.avg_question_score_perc.toFixed(3)}`
+                    : ''}
+                </td>
+                <td class="text-center">
+                  ${question.number_submissions_hist
+                    ? html` <div id="attemptsHist${iRow}" class="miniHist"></div> `
+                    : ''}
+                </td>
+                <script>
+                  $(function () {
+                    var data = [${question.number_submissions_hist}];
```

Comment:
Now might be a good time to change how this is done. In new code, we really want to avoid interpolating values into scripts. Instead, we'd typically attach data like this to a `data-` attribute (possibly as a JSON-encoded array or object) and then read it off the element in JS.

Reply:
Neat, TIL that jQuery's `.data(...)` will automatically deserialize JSON arrays/objects!

In the future, we could perhaps standardize on something like `data-histmini-values` and avoid the need to manually inject `<script>` tags for each one. No need to change that now though.